### PR TITLE
GB-14 smoking detection demo

### PR DESCRIPTION
**Pull request by mytcat**
screen shot 2018-05-03 at 4 51 26 pm

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,7 +38,21 @@ <h3>Extreme video innovation.</h3>
...
-		    <section class="page2" id="page2">
+			
+		    <section class="page2 section-no-border" id="page2">
+                <div class="page-2 overlay"></div>
+                <div class="section-logo"></div>
+				<div class="page-container">
+					<div class="hex hex-player-play"></div>
+					<div class="play-button-label"></div>
+					<h3>Do you need to detect nudity, profanity, smoking or any other standarts&practices in a video? Click play for smoking detection demo</h3>
+					<h1>STANDARTS & PRACTICES</h1>
```
Comment (A-Bush) [inline index.html]: `&` -> `&amp;`

File: index.html [hunk 2]
```diff
@@ -221,6 +237,117 @@ <h2 id="contact-headline">Hit Us Up</h2>
 		    </section>
 		</div>
 	</div>
+    <script>
+        var elements = {};
+        var pCont = document.querySelector("#playerContainer");
+        var videoUrl = "//kote.videogorillas.com/vmir/videogorillascom/smoking-dash/file.mpd";
+        if(/iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream) {
```
Comment (A-Bush) [inline index.html]: do we handle Safari users or not?

File: index.html [hunk 3]
```diff
@@ -212,7 +228,7 @@ <h5>The core team includes...</h5>
 					</div>
 				</div>
 		    </section>
-		    <section class="page6 section-no-border" id="page6">
+		    <section class="page7 section-no-border">
```
Comment (A-Bush) [inline index.html:231]: is it safe to delete id?

File: index.html [hunk 4]
```diff
@@ -221,6 +237,117 @@ <h2 id="contact-headline">Hit Us Up</h2>
 		    </section>
 		</div>
 	</div>
+    <script>
```
Comment (A-Bush) [inline index.html]: why script included in html and is not separate .js file?

File: index.html [hunk 5]
```diff
@@ -221,6 +237,117 @@ <h2 id="contact-headline">Hit Us Up</h2>
...
+                loadSubs();
+                loadAudio();
+                $('.vg_addon.vg_addon__list.vg_default-bg.vg_audiolist').on('mouseover', function (e) {
+                    $.fn.fullpage.setAllowScrolling(false);
+                });
+                $('.vg_addon.vg_addon__list.vg_default-bg.vg_audiolist').on('mouseleave', function (e) {
+                    $.fn.fullpage.setAllowScrolling(true);
+                });
+
+                let ocr = TimecodeOCRPlugin.init(elements.player.player, "newocr.tf/model.json");
```
Comment (A-Bush) [inline index.html]: does OCR works on video without timecodes?

**Review by A-Bush — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (mytcat) [reply index.html:231]: Yes. I have checked it before deleting.

File: index.html (hunk 2 above)
Comment (mytcat) [reply index.html]: We don't need to handle Safari. Just checking users who use Apple mobile devices.

**Review by A-Bush — APPROVED** [review]

**Pull request merged**